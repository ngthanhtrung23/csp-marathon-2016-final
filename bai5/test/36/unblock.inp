11123.
45523.
4006..
4.7688
..7AAB
ZZZ..B
