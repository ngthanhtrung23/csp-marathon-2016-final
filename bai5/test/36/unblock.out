32
3 D 1
Z R 1
4 D 2
0 L 1
5 L 1
7 U 1
A L 2
6 D 1
2 D 1
1 R 3
7 U 2
0 R 1
4 U 1
Z L 1
6 D 1
8 L 1
B U 3
8 R 1
6 U 1
Z R 1
4 D 1
0 L 1
7 D 1
1 L 3
2 U 1
3 U 1
6 U 1
A R 3
6 D 1
7 D 2
B U 1
0 R 4

